src/Lumper.UI/ViewModels/Tasks/ChangeTextureTaskViewModel.Item.cs

```diff
@@ -0,0 +1,67 @@
+﻿using ReactiveUI;
+using Lumper.Lib.Tasks;
+
+namespace Lumper.UI.ViewModels.Tasks;
+
+/// <summary>
+///     ViewModel for ChangeTextrue Task
+/// </summary>
+public partial class ChangeTextureTaskViewModel : TaskViewModel
+{
+    public abstract class ChangeTextureItem : ViewModelBase
+    {
+        public ChangeTextureItem(ChangeTextureTaskViewModel parent,
+                                 ChangeTextureTask task)
+        {
+            Parent = parent;
+            _task = task;
+        }
+
+        ~ChangeTextureItem()
+        {
+            Remove();
+        }
+
+        public ChangeTextureTaskViewModel Parent { get; }
+
+        protected readonly ChangeTextureTask _task;
+
+        protected string _source = ""; public string Source
+        {
+            get => _source;
+            set
+            {
+                if (value != _source && value is not null)
+                {
+                    Remove();
+                    _source = value;
+                    Add(Source, Target);
+
+                    this.RaisePropertyChanged(nameof(Source));
```

Comment:
From https://www.reactiveui.net/api/reactiveui/ireactiveobjectextensions/63fddd3b: "Leave null to let the runtime set to caller member name.". So you don't need the nameof, just this.RaisePropertyChanged() is fine.